### PR TITLE
Add cluster management section with API configuration

### PR DESCRIPTION
**Pull request by tkyi**
- added API Configuration page
- moved Plugins and Cluster Setup to new Cluster Management section as well
- updated some secret keys in Cluster Setup

Related to https://github.com/screwdriver-cd/screwdriver/issues/371

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -12,10 +12,11 @@ extra_javascript: [js/vendor/jquery-3.1.0.min.js, js/yaml.js]
 
 pages:
     - Home: 'index.md'
+    - Cluster Management:
+        - Set Up a Screwdriver Cluster: 'cluster-management/cluster-setup.md'
```
Comment (stjohnjohnson) [inline mkdocs.yml]: Isn't this only for running in Kubernetes?  Should we call this something related to that?

Comment (tkyi) [reply mkdocs.yml]: Yeah, that's true. Any suggestions?
`Set Up a Screwdriver Cluster in Kubernetes`
`Using Kubernetes with Screwdriver`
`Deploying Screwdriver API with Kubernetes`

Comment (tkyi) [reply mkdocs.yml]: My original idea was to have _all_ cluster setup in one page but it's too big already so it makes sense to keep this one Kubernetes-only

Comment (stjohnjohnson) [reply mkdocs.yml]: I would put it at the bottom of the list, maybe under a section called Examples and call it Kubernetes

Comment (tkyi) [reply mkdocs.yml]: Updated 👍

**Review by stjohnjohnson — APPROVED** [review]
LGTM 👍

**Pull request merged**